apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.tsx

```diff
@@ -59,6 +60,16 @@ function WelcomeTour() {
 		const paymentBlockIndex = tourSteps.findIndex( ( step ) => step.slug === 'payment-block' );
 		tourSteps.splice( paymentBlockIndex, 1 );
 	}
+	const { isInserterOpened, isSidebarOpened, isSettingsOpened } = useSelect( ( select ) => ( {
+		isInserterOpened: select( 'core/edit-post' ).isInserterOpened(),
+		isSidebarOpened: select( 'automattic/block-editor-nav-sidebar' ).isSidebarOpened(),
+		isSettingsOpened:
+			select( 'core/interface' ).getActiveComplementaryArea( 'core/edit-post' ) ===
+			'edit-post/document',
+	} ) );
+
+	const isTourMinimized =
+		isSidebarOpened || ( isMobile() && ( isInserterOpened || isSettingsOpened ) );
```

Comment:
I think Gutenberg uses the `MEDIUM` breakpoint for enabling the sidebars, which is at 782px. Could be helpful to use that instead of `isMobile` to ensure that this applies to all screens that would be affected by this.